pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java

```diff
@@ -231,12 +230,6 @@ private static void generateBrokerBasicMetrics(PulsarService pulsar, SimpleTextO
                 clusterName, Collector.Type.GAUGE, stream);
         }
 
-        if (pulsar.getConfiguration().isExposeManagedCursorMetricsInPrometheus()) {
```

Comment:
If using the Prometheus class to collect metrics, the metrics data will record automatically, right?

Reply:
yes